src/hooks/useDietTracker.js

```
@@ -0,0 +1,569 @@
+
+        // Refresh next meal
+        await fetchNextMeal();
+
+        return data;
+      } catch (err) {
+        console.error("Error completing meal:", err);
+        setError(err.message);
+        throw err;
+      }
+    },
+    [fetchNextMeal]
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)